proto/greptime/v1/ddl.proto

```diff
@@ -51,6 +51,12 @@ message DropTableExpr {
   string table_name = 3;
 }
 
+message FlashTableExpr {
+  string catalog_name = 1;
+  string schema_name = 2;
+  string table_name = 3;
```

Comment:
Should we also allow user to flush a specific region with an optional `region_id` like clickhouse's `OPTIMIZE TABLE ... PARTITION ...` clause?

Reply:
> Should we also allow user to flush a specific region with an optional `region_id` like clickhouse's `OPTIMIZE TABLE ... PARTITION ...` clause?

Done👏